ecosystem/sep-0038.md

```diff
@@ -7,14 +7,14 @@ Author: Jake Urban <@jakeurban> and Leigh McCulloch <@leighmcculloch>
 Track: Standard
 Status: Draft
 Created: 2021-04-09
-Updated: 2021-05-03
+Updated: 2021-06-28
 Discussion: https://github.com/stellar/stellar-protocol/issues/901
-Version 1.1.2
+Version 1.1.3
```

Comment:
It looks like this change adds fields to the responses as new functionality, so this version should be a minor version. i.e. `1.2.0`

Reply:
https://github.com/stellar/stellar-protocol/pull/987/commits/a727baf7a92e2e581a3fbd9c7295759b3a741605